app/dashboard/templates/ens/base.html

```diff
@@ -5,32 +5,17 @@
     <title>{% trans "ENS Subdomains | Gitcoin" %}</title>
     {% include 'shared/cards_pic.html' with title='Gitcoin | _you_.gitcoin.eth ENS Subdomains'  card_desc='*NEW* Register `_you_.gitcoin.eth` with one click for free' avatar_url="https://s.gitcoin.co/static/v2/images/ens_gitcoin.png"  %}
     {% include 'shared/head.html' %}
-    <meta charset="utf-8" />
-    <meta name="title" content="ENS Subdomains | Gitcoin">
-    <meta name="description" content="Create and manage your ENS subdomain">
-    <meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
-    <meta name="theme-color" content="#ffffff">
-    <!--[if lte IE 8]><script src="{% static "onepager/js/html5shiv.js" %}"></script><![endif]-->
-    <link rel="stylesheet" href="{% static "v2/css/bootstrap.min.css" %}" crossorigin="anonymous">
+
     <link rel="stylesheet" href="{% static "v2/css/kudos/styles.css" %}" />
     <link rel="stylesheet" href="{% static "onepager/css/main.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/lib/typography.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/gitcoin.css" %}">
-    <link rel="stylesheet" href="{% static "onepager/css/main.css" %}">
```

Comment:
wouldn't we need this ?

Reply:
`v2/css/lib/typography.css`
Already included in https://github.com/gitcoinco/web/blob/a6d073c3c8a3be999269e1ae0b390b13ff2cb814/app/retail/templates/shared/head.html#L36

`v2/css/gitcoin.css`
Already included in https://github.com/gitcoinco/web/blob/a6d073c3c8a3be999269e1ae0b390b13ff2cb814/app/retail/templates/shared/head.html#L38

`onepager/css/main.css`
The file is included twice, so I removed it once:

1. https://github.com/gitcoinco/web/blob/e8956ddd478e5aca3b3b4a134b40d0c6368d6f75/app/dashboard/templates/ens/base.html#L16
2. https://github.com/gitcoinco/web/blob/e8956ddd478e5aca3b3b4a134b40d0c6368d6f75/app/dashboard/templates/ens/base.html#L19

Reply:
Ah sweet